components/server/testd/Dockerfile

```diff
@@ -0,0 +1,4 @@
+
+
+FROM golang:1.17.2 as oci-tool-builder
+RUN go install github.com/csweichel/oci-tool@6f5c007
```

Comment:
```suggestion
RUN go install github.com/csweichel/oci-tool@v0.1.1
```

Reply:
Whoops, test file, removed it. Sorry for the noise. :pray: